braindecode/augmentation/functional.py

```diff
@@ -166,10 +166,12 @@ def _ft_surrogate(x=None, f=None, eps=1, random_state=None):
         device = f.device
     n = f.shape[-1]
     random_phase = _new_random_fft_phase[n % 2](
+        f.shape[0],
         n,
         device=device,
         random_state=random_state
     )
+    random_phase = torch.permute(torch.tile(random_phase, (f.shape[-2], 1, 1)), (1, 0, 2))
```

Comment:
I just wonder whether we want the same phase perturbation for all channels or not... Because this augmentation is based on the premise that Fourier phases don't matter, so in principle there should be no need to preserve phase dependence among channels. However, it might be good to only act on frequency information and leave spatial correlations intact. We can keep it like this for now. I'll run a few tests and we can create another PR it it seems better the other way around.